### DISCUSSION TITLE
Feedback on the New Iteration of the Syllabi Archive

### DISCUSSION THREAD

**Discussion by joshnatis** [RichAguil/HunterCS_CourseSyllabi#27] [Announcements]
A big re-design was introduced in https://github.com/Hunter-Open-Source-Club/HunterCS_CourseSyllabi/pull/26.

Please leave any questions, ideas, and opinions about it here.

Do you hate it? Do you love it? Did it threaten your cat? **Is it not working on your device?** Let us know.

**Comment by davemiyazaki** [comment]
Hey! The website is beautiful, and I am glad it keeps getting consistent updates( special note on the structure and flow of the animation). I have in mind one interesting suggestion to the  improvement, I am even ready to volunteer to provide all needed information and coding workforce. Among tons of ideas for the website, the one suggestion is to add the prerequisites on each of listed on the website courses. It would drastically help to have an approximate knowledge of what courses a student would able to take or not, and what courses to plan ahead. Let me know if you have any questions, would be glad to help!

**Reply by joshnatis** [reply]
Hey David, this was a really kind comment, I'm quite sorry I missed it :-(. I think this is a great idea if we could figure out a way to display the prerequisites in a non-obtrusive way -- I have a few ideas, I'm sure you did too. I think the simplest way would be to just show a "prerequisites" button when a course is hovered over, and when you click it it brings up a screen showing the prereqs. I'll think about this for now but let me know if you still wanted to work on it :), and if I can help. Thanks again for your warm comment

**Reply by davemiyazaki** [reply]
Hey Josh, thank you for your reply! If you don't mind my help, I would still want to help on in the progress. It doesn't seem a lot of work, so I am only in and ready to share some ideas too. Let me know if you want to collaborate on this together (if time and circumstances allow you) so we would use better way of contacting. Sincerely, thank you again!
